lsp-mode.el

```diff
@@ -6516,15 +6517,17 @@ TBL - a hash table, PATHS is the path to the nested VALUE."
 (defun lsp-configuration-section (section)
   "Get settings for SECTION."
   (let ((ret (ht-create)))
-    (mapc (-lambda ((path variable boolean?))
+    (mapc (-lambda ((path variable boolean? ignore-on-nil?))
```

Comment:
I am not sure if this has to be conditional - can we make this the default behavior and strip all properties that are nil and that are not `boolean?`.

Reply:
Make sense. Done